Mesh_3/include/CGAL/Mesh_3/polylines_to_protect.h

```diff
@@ -320,15 +333,21 @@ struct Angle_tester
       const typename Kernel::Point_3& p1 = g[v1].point;
       const typename Kernel::Point_3& p2 = g[v2].point;
 
-      if(CGAL::angle(p1, p, p2) == CGAL::ACUTE) {
-        // const typename Kernel::Vector_3 e1 = p1 - p;
-        // const typename Kernel::Vector_3 e2 = p2 - p;
-        // std::cerr << "At point " << p << ": the angle is "
-        //           << ( std::acos(e1 * e2
-        //                          / CGAL::sqrt(e1*e1)
-        //                          / CGAL::sqrt(e2*e2))
-        //                * 180 / CGAL_PI ) << std::endl;
-        return true;
+      if (!m_test_angle)
+      {
+        if (CGAL::angle(p1, p, p2) == CGAL::ACUTE)
+          return true;
+      }
+      else
+      {
```

Comment:
I suggest the following, to ensure that an acute angle will always `return true`:
```suggestion
      if (CGAL::angle(p1, p, p2) == CGAL::ACUTE)
        return true;
      if (m_test_angle)
      {
```